qiskit/chemistry/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uccsd_factory.py

```diff
@@ -111,8 +134,48 @@ def include_custom(self, include_custom: bool) -> None:
         """Setter of the ``include_custom`` setting for the ``expectation`` setting."""
         self._include_custom = include_custom
 
-    def get_solver(self,  # type: ignore
-                   transformation: FermionicTransformation) -> MinimumEigensolver:
+
+    @property
+    def method_singles(self) -> str:
+        """Getter of the ``method_singles`` setting for the ``method_singles`` setting."""
+        return self._optimizer
```

Comment:
```suggestion
        return self._method_singles
```

Reply:
We should have tests for these getters/setters!

Reply:
done.